packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman-drilldown.vue

```diff
@@ -819,10 +821,12 @@ async function renderCharts() {
 	updateStateCharts();
 	updateParameterCharts();
 
+	if (!validatedModelConfiguration.value) return;
+
```

Comment:
This function feels a bit odd and maybe some reorganization would be good? Are you actually wanting to return or do we want to throw an error? 

But moreover, the underlying model and MMT do not change across validation runs, so `getModel`/`getMMT` are likely better if they are in onMounted hook since they need to be loaded once and once-only. What is the reason we only get results and in "renderCharts"?

Reply:
I did the clean ups and I figured out a way to just process the funman results once. I was filtering the data depending on the "Show furthest results checkbox", but I decided to just mark which boxes are the furthest to use them later.